src/components/ItemControls/IconSelectionControls/IconSelectionControls.tsx

```diff
@@ -33,6 +33,13 @@ export const IconSelectionControls = () => {
     [mode, uiStateActions]
   );
 
+  useEffect(() => {
```

Comment:
I'm guessing this block is here to prevent simultaneous scrolling and zooming.  I have another suggestion on how to get around this further below.